css/style.css

```diff
@@ -15,5 +14,234 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+header,
+main,
+
+footer {
+    padding: 0 1.5rem;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    margin-top: 30px;
+}
+
+.nav-menu {
+    display: flex;
+    list-style: none;
+    flex-direction: row;
+    justify-content: end;
+
+}
+
+.nav-list {
+    margin-left: 15px;
+    padding: 0.rem 0;
+    font-family: 'sans-serif';
```

Comment:
If you look at the font in the nav bar on the header it looks quite diffrent from the font yours displays, so there's something wrong here. 

The problem is this line 'sans-serif' doesn't work because you have put it in quotes, so the browser looks for a font called 'sans-serif' instead of understanding it as the name of a group of fonts.

To fix, simply remove this line. If this doesn't make sense let me know.